pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "agntcy-app-sdk"
-version = "0.4.0"
+version = "0.4.0.dev0"
```

I think for complying with semantic versioning rules, we shall do this:
```suggestion
version = "0.4.0-dev.0"
```